digiroad2-api-oth/src/main/scala/fi/liikennevirasto/digiroad2/Digiroad2Api.scala

```diff
@@ -2264,18 +2274,39 @@ class Digiroad2Api(val roadLinkService: RoadLinkService,
         mapLightLane(usedService.getByZoomLevel( Some(LinkGeomSource.NormalLinkInterface)))
       } else {
         validateBoundingBox(boundingRectangle)
-        val assets = usedService.getByBoundingBox(boundingRectangle)
+        val (assets, roadLinksWithoutLanes) = usedService.getByBoundingBox(boundingRectangle, withWalkingCycling = params("withWalkingCycling").toBoolean)
```

Comment:
Shouldn't we do a getOrElse  as we do in zoom but to return false as default?

Reply:
Well we could but then you would need to do something like this
``` Scala
val withWalkingCycling = params.getOrElse("withWalkingCycling ", false).toString.toBoolean
```
And since we control the request and know that we always send as parameters "withWalkingCycling" is the getOrElse really necessary?

Reply:
It is just a validation/protection at backend side.
But if you think it is unnecessary.

Reply:
Done like this params.getAsOrElse[Boolean]("withWalkingCycling", false)